src/styles/card.css

```diff
@@ -0,0 +1,87 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background: var(--color-card-white);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+  --color-button-blue: #00acdc;
+  --color-card-white: #fff;
+  --color-text-dark: #060b35;
+  --color-text-grey: #616070;
+}
+
+.card__image {
+  height: 134px;
+  padding: 0 3px;
+  margin-bottom: 40px;
+}
+
+.card__product {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin-bottom: 4px;
+  color: var(--color-text-dark);
+}
+
+.card__code {
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--color-text-grey);
+  margin-bottom: 16px;
+}
+
+.card__reviews {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
```

Comment:
the same as on 51-53 lines, I suggest to move those properties into common separated class

Reply:
it's too much )